sdk/src/utils/ipfsClient.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const getContent = async (ipfsHash: string): Promise<string> => {
+  // Use the Cloudflare IPFS gateway URL
+  const ipfsGatewayUrl = `https://cloudflare-ipfs.com/ipfs/${ipfsHash}`;
+  // Make a request to the IPFS gateway
+  const response = await axios.get(ipfsGatewayUrl);
```

Comment:
Maybe we should handle potential errors (network, timeout, etc.) here, wdyt?

Reply:
Actually I thought about logging it, but it is used only to get content where we are not handling any error just to avoid so much noise, as get operations can be more in numbers. Also if ipfs call fails we are just catching it on one layer above and adding error message to offchain payload object.